lib/google-id-token.rb

```diff
@@ -132,7 +132,9 @@ def check_cached_certs(token, aud, cid)
       end
 
       if payload
-        if !(payload.has_key?('aud') && payload['aud'] == aud)
+        unless payload.has_key?('aud') &&
```

Comment:
I'd suggest simplifying this code a bit: 

```suggestion
        if !(payload.has_key?('aud') && Array(payload['aud']).include?(aud))
```

This way we always operate on an array.

```ruby
>> Array(["abc"])
["abc"]
>> Array("abc")
["abc"]
```